doc/api/cli.md

```diff
@@ -492,6 +441,87 @@ file will be created if it does not exist, and will be appended to if it does.
 If an error occurs while attempting to write the warning to the file, the
 warning will be written to stderr instead.
 
+### `--report-directory=directory`
+<!-- YAML
+added: v11.8.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/27312
+    description: changed from `--diagnostic-report-directory` to
```

Comment:
It seems other descriptions begin with upper case, so here and in all relevant YAML blocks below:
```suggestion
    description: Changed from `--diagnostic-report-directory` to
```